lib/aws_ssm_provider.ex

```diff
@@ -2,7 +2,7 @@ defmodule AwsSsmProvider do
   @moduledoc """
   Populate run time application variables from a file produced by a call to AWS SSM
   """
-  use Mix.Releases.Config.Provider
+  use Distillery.Releases.Config.Provider
```

Comment:
@JamesLavin  Thanks for the PR. I think we should bump the version of the project in mix.exs since this seems to be a breaking change.

Reply:
Definitely. And I'll add a note to the README.